CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Add `pipx install --preinstall` to support preinstalling build requirements
 - Pass `--no-input` to pip when output is not piped to parent stdout
 - Fix program name in generated manual page
+- Expose package manual pages
```

Comment:
```suggestion
- Expose manual pages included in an application installed with `pipx install`
```

Also we’d want to add some documentation on where a package should put the man files for pipx to find. Python does not actually have a standard on this, and current packages mostly do this by convention, so we need to describe the logic a bit for package maintainers to follow.

Reply:
I have added some documentation in bd6c65e.

The convention comes from the [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) and the [Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#usrsharemanManualPages).